Fix #35 and similar bugs. Refactor plivoxml

Abhishek-plivo opened the pull request:
- refactor plivo xml module and make it available as both plivo.xml and plivo.plivoxml
- Fix test modules and increase coverage
- Fix #35 and similar bugs

plivo-sdks commented on plivo/__init__.py:
```diff
@@ -1,4 +1,4 @@
 from . import exceptions
 from .rest import Client as RestClient
-import xml as plivoxml
+from . import xml as plivoxml
```
Can you make this `import .xml as plivoxml`?

plivo-sdks commented on plivo/rest/client.py:
```diff
@@ -171,7 +172,7 @@ def create_request(self, method, path=None, data=None):
 
     def send_request(self, request, **kwargs):
-        return self.session.send(request, **kwargs)
+        return self.session.send(request,timeout=self.timeout, **kwargs)
```
Add space after `request,`

Abhishek-plivo commented:
Changes after the review
- fixed linting using YAPF, PEP and isort
- reverted the timeout customization
- Fixed linting for test classes
- fixed linting of setup.py
